frontend/src/components/regions/CardDetails/index.js

```diff
@@ -145,7 +135,7 @@ function CardDetailsUnconnected({
     projectReviews: currentProjectReviews,
 
     handleClickAddReview,
-    showAddReviewButton,
+    getShowAddReviewButton,
```

Comment:
Dont pass thew function in. Call the function and pass the result into the component.